vlsi/example-nangate45.yml

```diff
@@ -0,0 +1,144 @@
+# Technology Setup
+# Technology used is nanagate45
+vlsi.core.technology: nangate45
```

Comment:
We might want to separate out the yaml into generic settings and tech specific settings, so we would have an:
example.yml
example-asap7.yml
example-nangate45.yml

And it optionally includes the right pair based on `tech_name`